builtin/credential/cert/path_certs.go

```diff
@@ -234,16 +235,9 @@ func (b *backend) pathCertWrite(ctx context.Context, req *logical.Request, d *fr
 		}
 	}
 
-	var parsedCIDRs []*sockaddr.SockAddrMarshaler
-	for _, v := range d.Get("bound_cidrs").([]string) {
-		parsedCIDR, err := sockaddr.NewSockAddr(v)
-		if err != nil {
-			if b.Logger().IsDebug() {
-				b.Logger().Debug(fmt.Sprintf("unable to parse %s as a cidr: %s", v, err))
-			}
-			return logical.ErrorResponse(fmt.Sprintf("unable to parse %s as a cidr", v)), logical.ErrInvalidRequest
-		}
-		parsedCIDRs = append(parsedCIDRs, &sockaddr.SockAddrMarshaler{parsedCIDR})
+	parsedCIDRs, err := parseutil.ParseAddrs(d.Get("bound_cidrs"))
+	if err != nil {
+		return logical.ErrorResponse(err.Error()), nil
```

Comment:
This should still return `logical.ErrInvalidRequest`

Reply:
ErrInvalidRequest is the default!

https://github.com/hashicorp/vault/blob/master/logical/response_util.go#L90

Reply:
That was my oversight!

I don't think it would default to `ErrInvalidRequest` because here it's under a `case` statement rather than a `default`. https://github.com/hashicorp/vault/blob/master/logical/response_util.go#L106

Reply:
It would, because it starts at ErrInvalidRequest. If nothing modifies it, it will stay there. Since in this case nothing is modifying it...